packages/discovery-react-components/src/components/StructuredQuery/components/RuleRow/RuleRow.tsx

```diff
@@ -0,0 +1,52 @@
+import * as React from 'react';
+import { ComboBox, TextInput } from 'carbon-components-react';
+import { Messages } from 'components/StructuredQuery/messages';
+import { structuredQueryRulesClass } from 'components/StructuredQuery/cssClasses';
+
+export interface RuleRowProps {
+  /**
+   * override default messages for the component by specifying custom and/or internationalized text strings
+   */
+  messages: Messages;
+}
+
+export const RuleRow: React.FunctionComponent<RuleRowProps> = ({ messages }) => {
+  const handleOnFieldChange = () => {
+    // TODO: Fully implement handling field dropdown selections in a future issue
+  };
+
+  const handleOnOperatorChange = () => {
+    // TODO: Fully implement handling operator dropdown selections in a future issue
+  };
+
+  return (
+    <div className={structuredQueryRulesClass}>
+      <ComboBox
+        id="structured-query-rules-field-0"
```

Comment:
for this ID should we be passing it down or randomizing it? what happens if it isn't provided? this would potentially create issues if more than one `RuleRow` is on the page (which im imagining there will be)

Reply:
Yeah, I'm handling this more in the next issue where we actually have multiple `RuleRow`s on the page. If it's not provided, we get a warning since it's required. Basically, my implementation in the next issue so far is to use the `id` of the rule row, which will be part of what's passed down. I figured it was okay to leave this way at this point since there was no way to add another rule row so far.